static/app/components/replays/videoReplayerWithInteractions.tsx

```diff
@@ -121,6 +124,26 @@ export class VideoReplayerWithInteractions {
       }
     });
 
+    // log instances where we have a pointer touchStart without a touchEnd
+    const touchEvents = eventsWithSnapshots.filter(
+      (e: eventWithTime) => isTouchEnd(e) || isTouchStart(e)
+    );
+    const grouped = Object.groupBy(touchEvents, (t: any) => t.data.pointerId);
+    Object.values(grouped).forEach(t => {
+      if (t?.length !== 2) {
+        Sentry.captureMessage(
+          'Mobile replay has mismatching touch start and end events',
+          {
+            tags: {
+              sdk_name: context.sdkName,
+              sdk_version: context.sdkVersion,
+              type: typeof t,
```

Comment:
`type` is quite generic, might conflict with something, how about:

```suggestion
              event_type: typeof t,
```